### PR TITLE
gunzip and null encoding

### LINKED ISSUE #66: gunzip real-world fixtures?
The real-world fixtures are all gzipped (which is confusing since they don't include `.gz` extensions), so there are a few ideas here:

👟  unzip them all and store as raw buffers in the repo/module (increases module size)
👞  update file extension, that's it 
👡  update file extension, build in function (at least for JS downstreams) to get a real world buffer unzipped

I'm in favor of 👟 

cc @mapbox/core-tech

### PR DESCRIPTION
**Pull request by mapsam**
This resolves #66 by gunzipping all fixtures in the real-world/ directory. One last step is to create a new fixture set with intentionally gzipped fixtures per @joto's comment in that issue.

Directory size before/after - `5.3 MB` / `8.4 MB` - expected since we are uncompressing.

cc @mapbox/core-tech

### CODE REVIEW

**Comment by mapsam** [comment]
Just added four `.mvt.gz` compressed fixtures! Ready for a review if anyone has a moment 👀

File: REAL-WORLD.md [hunk 1]
```diff
@@ -6,3 +6,4 @@ nepal|terrain|mapbox.mapbox-streets-v7,mapbox.mapbox-terrain-v2|z13|`real-world/
 uruguay|complex water feature|mapbox.mapbox-streets-v7,mapbox.mapbox-terrain-v2|z9|`real-world/uruguay/{z}-{x}-{y}.mvt`|12
 bangkok|urban and suburban|mapbox.mapbox-streets-v7,mapbox.mapbox-terrain-v2|z12|`real-world/bangkok/{z}-{x}-{y}.mvt`|40
 norway|many little islands|mapbox.mapbox-streets-v7,mapbox.mapbox-terrain-v2|z12|`real-world/norway/{z}-{x}-{y}.mvt`|32
+compressed|gzip compressed tiles in a light urban area|mapbox.mapbox-streets-v7|z14|`real-world/compressed/{z}-{x}-{y}.mvt`|4
```
Comment (springmeyer) [inline REAL-WORLD.md]: Should this be `{z}-{x}-{y}.mvt.gz`?

File: scripts/get-real-world.js [hunk 2]
```diff
@@ -18,7 +19,8 @@ for (let e in extents) {
 
   for (let x = xyz.minX; x <= xyz.maxX; x++) {
     for (let y = xyz.minY; y <= xyz.maxY; y++) {
-      q.defer(getAndWriteTile, e, extents[e].tileset, extents[e].zoom, x, y);
+      // q.defer(getAndWriteTile, e, extents[e].tileset, extents[e].zoom, x, y);
```
Comment (springmeyer) [inline scripts/get-real-world.js]: very minor: remove this commented line?

**Comment by mapsam** [comment]
Thanks @springmeyer! Merging away.

**Pull request merged**